README.md

```diff
@@ -96,4 +99,37 @@ To generate a new encrypted value or to replace one of the provided values, for
 $ ansible-vault encrypt_string --vault-password-file your_password_file --name 'trello_api_key'
 ```
 
-then enter the value you want to encrypt when prompted.
\ No newline at end of file
+then enter the value you want to encrypt when prompted.
+
+## Docker
+
+Run the project:
+```bash
+docker build --tag todo-app .
+docker run -dit \
+    --name todo-app \
+    -p 8000:8000 \
+    --env-file .env \
+    todo-app
+```
+
+Run the project with mounting:
+```bash
+docker build --target development --tag todo-app:dev .
+docker run -dit \
+    --name todo-app:dev \
+    -p 8000:8000 \
+    --env-file .env \
+    --mount type=bind,source="$(pwd)/todo_app",target=/app/todo_app,readonly \
+    todo-app-dev
+```
+
+Run the project in production environment:
+```bash
+docker build --target production --tag todo-app:prod .
+docker run -dit \
+    --name todo-app:prod \
+    -p 8000:8000 \
+    --env-file .env \
+    todo-app-prod
```

Comment:
Similarly, we need to provide the name of the _image_ as the last argument here, which needs to match your build command!
```suggestion
    todo-app:prod
```